src/script/util/KeyboardUtil.ts

```diff
@@ -78,7 +78,10 @@ export const isMetaKey = (keyboardEvent: KeyboardEvent): boolean =>
 export const isPasteAction = (keyboardEvent: KeyboardEvent): boolean =>
   isMetaKey(keyboardEvent) && isKey(keyboardEvent, KEY.KEY_V);
 
-export const isRemovalAction = (key: string): boolean => [KEY.BACKSPACE, KEY.DELETE].includes(key);
+const removalKeys: string[] = [KEY.BACKSPACE, KEY.DELETE];
+export const isRemovalAction = (key: string): boolean => removalKeys.includes(key);
+
+export const isSpaceOrEnterKey = (key: string): boolean => key === KEY.SPACE || key === KEY.ENTER;
```

Comment:
I wanted to do this, but TS doesn't like it because

`Argument of type 'string' is not assignable to parameter of type '"Enter" | " "'.ts(2345)`, 

so I did it like this not to cast the type ;)